src/test/regress/expected/shard_rebalancer.out

```diff
@@ -178,9 +178,17 @@ SELECT pg_sleep(.1); -- wait to make sure the config has changed before running
 
 (1 row)
 
-SET citus.shard_replication_factor TO 2;
```

Comment:
I feel like propagating a setting that's created outside the transaction block is actually a good test. Can we add one back?